vault/package/install.sls

```diff
@@ -51,8 +53,14 @@ vault-package-install-file-symlink:
     - target: /opt/vault/bin/vault
     - force: true
 
+vault-package-install-pkg-installed:
+  pkg.installed:
+    - name: {{ vault.setcap_pkg }}
```

Comment:
Since `setcap_pkg` is only defined in `osfamilymap.yaml`, this will result in an error for OS families that this hasn't been defined for.  So either an `if` block here to check it is available in the map, or supplying a value in `defaults.yaml` (even an empty string?).

Reply:
Good catch